README.md

```diff
@@ -1,5 +1,15 @@
 # GOV.UK coronavirus - find support
 
+⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️
```

Comment:
Nit: I think this is a bit much - maybe a simple addition to the header would do

# GOV.UK coronavirus - find support (ARCHIVED ⚠️)

Reply:
@theseanything - thanks - fixed